profanityfilter/profanityfilter.py

```diff
@@ -53,9 +53,35 @@ def append_words(self, word_list):
         """Define a custom list of profane words to be used in conjunction with the default list."""
         self._extra_censor_list.extend(word_list)
 
-    def remove_word(self, word):
-        """Remove given word from censor list."""
-        self._censor_list.remove(word)
+   def remove_word(self, word, anywhere=True):
+        """Remove given word from censor list.
+
+        Anywhere:
+            - if True, remove word from extra_censor_list if word is there
+            - if False, only look in default/custom censor list"""
+        if word in self._extra_censor_list and anywhere:
+            self._extra_censor_list.remove(word)
+        else:
+            if self._custom_censor_list: #NEW - now supports word removal from custom list, if applicable - saves defining new lists to remove words from custom list
+                self._custom_censor_list.remove(word)
+            else:
+                self._censor_list.remove(word) #This is the only place the code would check previously
+                    
+     def remove_word_list(self, word_list, anywhere=True):
```

Comment:
```suggestion
     def remove_words(self, word_list, anywhere=True):
```

Reply:
Thank you for this one - the reason I did word_list was to avoid confusion between similar names for methods, but I agree this makes more sense